codility_training/lessons.lesson03.TimeComplexity.TapeEquilibrium/eyabc.js

```diff
@@ -0,0 +1,16 @@
+function solution(A) {
+  let frontSum = 0;
+  let backSum = A.reduce((sum, value) => sum + value);
+  const len = A.length - 1;
+  return A.reduce((sum, v, k) => {
+    if (k === len) return sum;
+    frontSum += v;
+    backSum -= v;
+    const diff = Math.abs(frontSum - backSum);
+    if (k === 0 || diff < sum) return diff;
```

Comment:
9~10 까지는 다음과 같이 변경할 수 있음
```suggestion
    return Math.min(Math.abs(frontSum - backSum), min)
```